README.md

```diff
@@ -378,6 +376,108 @@ To delete your test dataset, use the script [`fair-api-datasets-delete.py`](fair
 python fair-api-datasets-delete.py <dataset_code>
 ```
 
+## Dictionaries
+
+As above, ensure that the `FAIR_API_ENDPOINT` and `FAIR_API_TOKEN` are set in your environment.
+
+### List all dictionaries
+
+Run:
+
+```sh
+python fair-api-dictionaires-list.py
+```
+
+Result:
+
+```text
+Found 74 dictionaries
+
+...
+athlete_events - athlete_events
+climate70 - climate70
+golden_globe_awards - golden_globe_awards
+...
+```
+
+### Get single dictionary
+
+Run:
+
+```sh
+python fair-api-dictionaries-get.py <code>
+```
+
+Result:
+
+```text
+{
+  "id": 127,
+  "name": "synthetic_alzheimers_profile",
+  "code": "synthetic_alzheimers_profile",
+  "description": null,
+  "created_at": "2021-04-21T15:57:16.968Z",
+  "updated_at": "2021-11-09T15:00:08.262Z",
+  "fields": [
+    {
+      "name": "study_id",
+      "label": "Study ID",
+      "description": "Participant Study ID",
+      "constraints": "STUDY_ID",
+      "uri": null,
+      "pseudonymisation_rule": null,
+      "type": "text"
+    },
+    {
+      "name": "site",
+      "label": "Site",
+      "description": "Trial Delivery Centre",
+      "constraints": "SITE",
+      "uri": null,
+      "pseudonymisation_rule": null,
+      "type": "text"
+    },
+    ...
+  ],
+  "lookups": {},
+  "dataset_id": 133
+}
+```
+
+### Update a Dictionary
```

Comment:
Do we need a section for updating a catalogue too?

Reply:
Catalogues are part of the dataset update for now. Given the catalogue templating feature won't be released this year I'm happy to leave out those changes until then.